packages/ui/src/components/va-virtual-scroller/VaVirtualScroller.demo.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <VbDemo class="va-virtual-scroller-demo">
+    <VbCard title="default">
+      <va-virtual-scroller
```

Comment:
Default demo should have absolute minimum of props defined. From my perspective, that's `items` and `v-slot`